plugins/python-build/bin/python-build

```diff
@@ -933,7 +933,7 @@ build_package_micropython() {
   { cd mpy-cross
     "$MAKE" $MAKE_OPTS
     cd ../ports/unix
-    "$MAKE" $MAKE_OPTS axtls
+    grep -qwF axtls Makefile && "$MAKE" $MAKE_OPTS axtls
```

Comment:
This is ugly, I don't know if there's a standard way of handling similar things in pyenv. On the other hand I wonder if there was a way to avoid doing this explicitly in the first place.

This might also break earlier version builds; tested with 1.15 that it doesn't.

Reply:
If the build procedure has changed, the standard way is to make another build_* function (in this case, build_package_micropython) and call the correct function for corresponding versions. We already did that for another flavour.

Reply:
Turns out that the explicit `axtls` make seems to have been unnecessary since 1.10. Duplicating the function seems a bit overkill, as we can use the package name for good enough results IMO. Modified the PR as such.